CONTRIBUTING.md

```diff
@@ -12,9 +12,54 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-#Adding a new section
+## Permissions & Contributions
 
-##Structure/Pattern
+There are two ways in which you can help update the content:
+
+* **One-off:** \
+If you are not a regular contributor to the project, but you would like to contribute some changes, the best way to do it is:
+
+  1. Fork the repo
+  1. Create a PR
+
+* **Periodic and regular contributions:** \
+If you plan to update the content semi-regularly or regularly, you can be added to the project's Contributors group. Please contact one of the project's administrators to be added to the group.
+
+## Git guidance
+
+Consistent with the practices suggested in this playbook, please follow the specifics regarding git as described in this section.
+
+### Branch naming convention
+
+In this repo, we will the following branch naming conventions:
+
+| Branch Type       | Pattern     | Example     |
+| ------------- | ---------- | ----------- |
+|  Feature | feature/\<issue#>-\<short description> | feature/498-reorganize-scm-section |
+| Bug Fix | fix/\<bug#>-\<short description> | bug/978-correct-grammar-myfile.md |
+| Personal | users/\<alias>/\<short description> | johndo/rewrite-grooming-section
```

Comment:
To make it simple, we could just skip the personal branches and the recommendation to not use personal branches

Reply:
I like it - I just added it because I saw a couple of them, and they are mentioned in the guidance. I removed and modified the note